resources/assets/sass/components/_gallery.scss

```diff
@@ -71,3 +70,299 @@
+
+        svg {
+
+            @include color-modifiers-and('fill');
+
+            fill: $color-white;
+            height: $spacer * 2;
+            width: $spacer * 2;
+        }
+
+        &:hover {
+
+            svg {
+
+                fill: $color-white;
+            }
+        }
+    }
+
+    .c-gallery__modal-inner {
+
+        position: absolute;
+        left: 0;
+        top: 0;
+        height: 100%;
+        width: 100%;
+
+        @include respond-to(tablet) {
+
+            @include wrap();
+
+            left: 50%;
+            top: 50%;
+            height: 70%;
+            transform: translate(-50%, -50%);
+        }
+
+        @include respond-to(desktop) {
+
+            margin-top: -$spacer * 6;
+        }
+
+        @include respond-to(desktop-medium) {
+
+            margin-top: -$spacer * 6.5;
+        }
+
+        @include respond-to(desktop-large) {
+
+            margin-top: -$spacer * 7;
+        }
+    }
+
+        .c-gallery__modal-image-container {
+
+            @include color-modifiers-dark-and('background');
+
+            position: relative;
+            width: 100%;
+            height: 100%;
+        }
+
+            .c-gallery__modal-image {
+
+                display: none;
+                height: 100%;
+                width: 100%;
+                box-shadow: 0 12px 36px $color-fill-black;
+
+                 @each $name, $color, $color-dark in $colors {
+
+                    .c-gallery__modal.m-#{$name} & {
+
+                        box-shadow: 0 12px 36px $color-dark;
+                    }
+                }
+
+                &.m-active {
+
+                    display: block;
+                }
+
+                img {
+
+                    position: relative;
+                    top: 50%;
+                    left: 50%;
+                    max-width: 100%;
+                    max-height: 100%;
+                    width: auto;
+                    height: auto;
+                    transform: translate(-50%, -50%);
+
+                    @include respond-to(desktop) {
+
+                        object-fit: cover;
+                        height: 100%;
+                        width: 100%;
+                    }
+                }
+            }
+
+                .c-gallery__modal-tags {
+
+                    position: absolute;
+                    left: $spacer;
+                    bottom: $spacer;
+                    display: none;
+
+                    @include respond-to(desktop) {
+
+                        display: block;
+                        max-width: $spacer * 40;
+                    }
+                }
+
+                .c-gallery__modal-title {
+
+                    color: $color-white;
+                    font-size: $font-size-small;
+                    text-align: right;
+                    position: absolute;
+                    right: $spacer;
+                    bottom: $spacer;
+                    text-shadow: 0px 1px 3px rgba($color-text-black, .5);
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>`0px` should be written without units as `0`